scout/server/blueprints/variants/templates/variants/variants.html

```diff
@@ -151,6 +151,26 @@ <h4 class="panel-title">
       <span class="glyphicon glyphicon-comment"></span>
     </a>
   {% endif %}
+
+
+  {% if variant.panels|count > 0 %}
```

Comment:
Great for the research view! For clinical use, we essentially always see variants in multiple panels - e.g. the primary analysis panel, OMIM and OMIM-AUTO? Do you see a need for the clinical side as well? Otherwise perhaps check if this is a research view and then display?